### DISCUSSION TITLE
7.0.6version: -font simsun.ttc -fontsize 17 will crash

### DISCUSSION THREAD

**Discussion by zhouleiMax** [ImageMagick/ImageMagick#5752] [Development]
7.0.6version: -font simsun.ttc -fontsize 17 will crash
use the ImageMagick:
./convert ./34x18.png -fill red -stroke red -font simsun.ttc -pointsize 17 -gravity center -annotate +0+0 '获奖' dt.png
the result is aborted

**Comment by dlemstra** [comment]
Can you share your input image and font file? And can you reproduce it with the latest version?. Can you test this in our GitHub Codespace.

**Reply by zhouleiMax** [reply]
hello, i find that all input file will crash, you can try, the example is a image that it's width == 34 pix, height == 18 pix.   font file is simsun.ttc,

**Reply by zhouleiMax** [reply]
I haven't tried it with the latest version， because I don't seem to see anyone asking this question

**Reply by zhouleiMax** [reply]
the matter is the font(simsun.ttc) and fontsize(17), sorry,  i hasn't test this in GitHub Codespace

**Reply by zhouleiMax** [reply]
[simsun.zip](https://github.com/ImageMagick/ImageMagick/files/9997075/simsun.zip)

**Reply by zhouleiMax** [reply]
![34x18](https://user-images.githubusercontent.com/103361315/201519679-864cd456-03f2-4b7b-af68-d4e6691a694c.png)

**Reply by zhouleiMax** [reply]
hello，i use the lastest version:
./magick ./34x18.png -fill red -stroke red -font simsun.ttc -pointsize 17 -gravity center -annotate +0+0 '获奖' dt.png
can't get the result~

**Reply by dlemstra** [reply]
I cannot reproduce your issue with the latest version of ImageMagick in our GitHub Codespace.

**Reply by zhouleiMax** [reply]
can you show me the result image? thank you , because i can't get the correct image when i use the command.

**Reply by zhouleiMax** [reply]
when I use the lastest version , I get the result is that 
![dd](https://user-images.githubusercontent.com/103361315/201627912-0befc0b0-ce92-4c1f-a44e-3cf5ca61c85d.png)

**Comment by dlemstra** [comment]
I am getting this output image: ![dt](https://user-images.githubusercontent.com/10426229/201720258-ff0c9d89-3de5-40b3-90e4-90f5c3332c83.png)

**Reply by zhouleiMax** [reply]
thank you ！
